.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
```

Comment:
Can we use spaces instead of tabs in this file as that’s what we do across the rest of the repo?